src/app/pages/forms-and-validation/verify-on-submit/slide-panel/slide-panel.component.scss

```diff
@@ -0,0 +1,14 @@
+:host {
+    display: block;
+    overflow: hidden;
+}
+
+.panes {
+    height: 100%;
+    width: 200%;
```

Comment:
On a windows device, does this cause any weird scrollbars to appear during the animation?  Mac looks fine.

Reply:
It is not showing scrollbars on windows.